iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
+mock_config: "./extras/centOS/7.2/mock/spin-iso-epel-7-ppc64le.cfg"
+rpm_dir: "./results"
+kickstart_file: "host-os.ks"
+hostos_package_group: "virtualization-hypervisor"
+package_group_list:
```

Comment:
Short answer is not, the reason for this group list is that it came from official CentOS 7 ppc64le ISO, however, I've made it flexible exactly for this scenario, i.e., hostos decided to provide a custom CentOS ISO instead of the official ISO + hostos packages.

If the team prefers I can use a custom list instead of the official one. If this is the case we can discuss a minimum group list.